sailfish/README.md

```diff
@@ -0,0 +1,37 @@
+# Kaidan Sailfish adaptions
```

Comment:
I am sure we getting more instructions for more plattforms. In my opinion we should keep the overall Readme small for the default linux version. And handle the other ones in separat files.